# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaManager.cpp
# Commit: 3aef6c404e07
# Full Hash: 3aef6c404e07742214a5e3c1ccbcbd2ad1eee959
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-28 04:41:16
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1742655 - Rename AudioInputTrack to AudioProcessingTrack r=pehrsons
#   
#   Depends on D134628
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D134629
# ==============================================================================

diff -r b004b579d1ce -r 3aef6c404e07 dom/media/MediaManager.cpp
--- a/dom/media/MediaManager.cpp	Thu Jan 27 22:29:35 2022 +0000
+++ b/dom/media/MediaManager.cpp	Thu Jan 27 22:29:35 2022 +0000
@@ -1530,7 +1530,7 @@
       if (mAudioDevice->IsFake()) {
         track = mtg->CreateSourceTrack(MediaSegment::AUDIO);
       } else {
-        track = AudioInputTrack::Create(mtg);
+        track = AudioProcessingTrack::Create(mtg);
         track->Suspend();  // Microphone source resumes in SetTrack
       }
 #else